L/libsingular_julia/common.jl

```diff
@@ -4,21 +4,16 @@ using BinaryBuilder
 import Pkg: PackageSpec
 
 const name = "libsingular_julia"
-const version = v"0.2.0"
+const version = VersionNumber(0, 2, julia_version.minor)
```

Comment:
Given the new versioning scheme started in #2234 it might make sense to use a variant of that here, too